### PR TITLE
Multiple Tables and Primary Key/Foreign Key Features

### PR DESCRIPTION
**Pull request by Mar12358**
# Added Features:
- Update Readme file.
- Multiple Tables, Primary Key and Foreign Key Features.

### Create a tables owners and species:
image

### Modify animals table:
  - Make sure that id is set as autoincremented PRIMARY KEY
image
  - Remove column `species`
  - Add column `species_id` which is a foreign key referencing species table
  - Add column `owner_id` which is a foreign key referencing the `owners` table
image

### Insert data into the owners' and species' table:
image
image

### Modify the inserted animals so it includes the species_id value:
image

### Modify the inserted animals so it includes the owners_id value:
image

### Write queries (using `JOIN`) to answer the following questions:
- What animals belong to Melody Pond?
image
- List of all animals that are Pokemon:
image
- List all owners and their animals, remember to include those that don't own any animal:
image
- How many animals are there per species?
image
- List all Digimon owned by Jennifer Orwell:
image
- List all animals owned by Dean Winchester that haven't tried to escape.
image
- Who owns the most animals?
image

### CODE REVIEW

**Review by rloterh — COMMENTED** [review]
Hi Mar, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

Kindly ensure that your ReadMe is professionally documented; Ensure that provided placeholder texts/ guide instructions are removed and/or replaced with appropriate contents applicable under each section of your ReadMe.

**Kindly remove/update all aspects similar to the underlined:**
image

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: queries.sql [hunk 1]
```diff
@@ -59,4 +59,32 @@ GROUP BY species;
...
+SELECT a.name FROM animals a
+JOIN species s ON a.species_id = s.id
+JOIN owners o ON a.owners_id = o.id
+WHERE s.name ='Digimon' AND o.full_name = 'Jennifer Orwell'
+
+SELECT a.name FROM animals a
+JOIN owners o ON a.owners_id = o.id
+WHERE a.escape_attempts = 0 AND o.full_name = 'Dean Winchester'
+
+
```
Comment (rloterh) [inline queries.sql]: - [x] Nice job demonstrating the screenshot for the query: _**Who owns the most animals?**_ Most importantly however, let's ensure that this query also present in the `queries.sql`

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Mar, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project, but you are almost there! 
🏁 🚦 🏃 🌴 🏫 🌳 🏘️

## Highlights 👍  
- JOIN queries properly implemented so far!

## Required Changes ♻️
_Kindly check the inline comments under the review._

Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 1 above)
Comment (Mar12358) [reply queries.sql:89]: I forgot the last one. Thanks!

Comment (Mar12358) [reply queries.sql]: I forgot the last one. Thanks!

**Review by Davidosky007 — APPROVED** [review]
Hi @Mar12358 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
*  Primary key & foreign key mechanism for joining tables used properly 👍 
* Multiple table queried properly 👍  
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**